pkg/crc/cluster/cluster.go

```diff
@@ -287,7 +287,7 @@ func EnsurePullSecretPresentOnInstanceDisk(sshRunner *ssh.Runner, pullSecret Pul
 	if _, _, err := sshRunner.Run(fmt.Sprintf("test -e %s", vmPullSecretPath)); err == nil {
 		return nil
 	}
-	logging.Info("Adding user's pull secret to instance disk...")
+	logging.Info("Adding user's pull secret to instance disk ...")
```

Comment:
Really not sure what the goal is here? Consistency with the other log messages? Or something else?
```
$ git grep "[^ ]\.\.\.\"" :^vendor |wc -l
21

$ git grep " \.\.\.\"" :^vendor |wc -l
17
```

There are currently more 'no space before ...' messages than 'space before ...' messages.

Reply:
Yes the goal was consistency. I didnt check for others! Let s drop this commit.

Reply:
(this commit is still present)